package.json

```diff
@@ -63,43 +64,43 @@
     "release": "bumpp && npm publish"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.6.2",
-    "defu": "^6.1.2",
+    "@nuxt/kit": "^3.8.1",
```

Comment:
check also to revert this and check it with i18n edge version (should be using `#app`)